addons/mail/tests/discuss/test_discuss_channel_as_guest.py

```diff
@@ -110,3 +98,12 @@ def test_sidebar_in_public_page(self):
         channel_1._add_members(guests=guest)
         channel_2._add_members(guests=guest)
         self.start_tour(f"/discuss/channel/{channel_1.id}", "sidebar_in_public_page_tour", cookies={guest._cookie_name: guest._format_auth_cookie()})
+
+    def test_invitation_link_redirects_internal_users_to_discuss(self):
+        bob = mail_new_test_user(self.env, "bob", groups="base.group_user")
+        self.authenticate(bob.login, bob.login)
+        channel = self.env["discuss.channel"]._create_channel(name="Channel 1", group_id=None)
+        response = self.url_open(channel.invitation_url)
+        group = self.env["discuss.channel"]._create_group(name="Group 1", partners_to=bob.partner_id.ids)
+        response = self.url_open(group.invitation_url)
+        self.assertIn(f"/odoo/action-mail.action_discuss?active_id={group.id}", response.url)
```

Comment:
For consistency, we should have the same test for guests.

Reply:
Guest can't be redirected to discuss. And we already have standard guest page tours.

Reply:
I don't see the point in adding more tests, thus more code to maintain, not to increase the test coverage :thinking: